README.md

```diff
@@ -140,7 +140,7 @@ Additionally, Qemistree also supports the inclusion of structural annotations ma
 qiime qemistree make-hierarchy \
   --i-csi-results fingerprints.qza \
   --i-feature-tables feature-table.qza \
-  --i-ms2-matches /path-to-MS2-spectral-matches.qza/ \
+  --i-ms2-matches path-to-MS2-spectral-matches.qza/ \

```

Comment:
I think the previous version is fine for this. It's just convention to write /path-to-blah/ inside forward slashes.